### DISCUSSION TITLE
german translation error

### DISCUSSION THREAD

**Discussion by vquie** [open-webui/open-webui#3823] [General]
Hi, I am about to create a small PR with a small fix to the german translation. The PR says I need to create the discussion first.

The text `"Continue with {{provider}}"` currently translates to `"Mit {{Anbieter}} fortfahren"`

**Comment by vquie** [comment]
https://github.com/open-webui/open-webui/pull/3824

**Comment by justinh-rahb** [comment]
Appreciate that. Translations PRs we're pretty liberal with, feel free to make your correction, just be sure to target the `dev` branch and watch for conflicts.
